ibis/backends/tests/test_temporal.py

```diff
@@ -1432,6 +1427,11 @@ def test_interval_add_cast_column(backend, alltypes, df):
     raises=AttributeError,
     reason="'StringColumn' object has no attribute 'strftime'",
 )
+@pytest.mark.broken(
+    ["bigquery"],
+    raises=GoogleBadRequest,
+    reason="FORMAT_TIMESTAMP does not support for DATETIME arguments",
```

Comment:
```suggestion
    reason="FORMAT_TIMESTAMP does not support DATETIME arguments",
```

Is there a `FORMAT_DATETIME` function we could call instead?

Reply:
Done.